diff -bur libcocaine-core2-0.11.2.0.orig/CMakeLists.txt libcocaine-core2-0.11.2.0/CMakeLists.txt
--- libcocaine-core2-0.11.2.0.orig/CMakeLists.txt	2013-12-18 16:43:36.000000000 +0200
+++ libcocaine-core2-0.11.2.0/CMakeLists.txt	2014-01-14 00:44:31.309000000 +0200
@@ -120,8 +120,8 @@
 
 TARGET_LINK_LIBRARIES(cocaine-core
     archive
-    boost_system-mt
-    boost_filesystem-mt
+    boost_system
+    boost_filesystem
     ${LIBCGROUP_LIBRARY}
     ${LIBCRYPTO_LIBRARY}
     ev
@@ -139,7 +139,7 @@
 
 TARGET_LINK_LIBRARIES(cocaine-runtime
     ${LIBBFD_LIBRARY}
-    boost_program_options-mt
+    boost_program_options
     cocaine-core)
 
 SET_TARGET_PROPERTIES(cocaine-core cocaine-runtime PROPERTIES
